### PR TITLE
feat: Add basic fastify docs

### PR DESCRIPTION
**Pull request by mydea**
This is a bit WIP, but you can see where this is going.

### CODE REVIEW

File: docs/platforms/javascript/guides/fastify/performance/instrumentation/opentelemetry.mdx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+```js
+const Sentry = require("@sentry/node");
+const {
+  GenericPoolInstrumentation,
+} = require("@opentelemetry/instrumentation-generic-pool");
+const { registerInstrumentations } = require("@opentelemetry/instrumentation");
+
+Sentry.init({
+  dsn: "__DSN__",
+});
```
[the remark this answers is not in the document]
Comment (mydea) [reply docs/platforms/javascript/guides/fastify/performance/instrumentation/opentelemetry.mdx:22]: good point, adding it!

**Review by s1gr1d — APPROVED** [review]
Looks good now 👍🏻

File: docs/platforms/javascript/guides/fastify/index.mdx [hunk 2]
```diff
@@ -0,0 +1,77 @@
+---
+title: Fastify
+description: "Learn about using Sentry with Fastify."
+---
```
Comment (stephanie-anderson) [inline docs/platforms/javascript/guides/fastify/index.mdx:4]: Here we need to add the category information - have a look at the Node guide to see how it is set.

Comment (mydea) [reply docs/platforms/javascript/guides/fastify/index.mdx:7]: it worked 🚀

**Pull request merged**